.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+Please include a summary of the change and which issue is fixed or feature developed.
+
+Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Link to the Linear Issue: # (issue)
+(this can also be specified in the title of the PR)
+
+## How can (was) this tested?
+
+Please describe the tests that you ran to verify your changes.
+Provide instructions so we can reproduce.
+
+Please also list any relevant details for your test configuration:
```

Comment:
I think unit tests should work out of the box so maybe this line isn't that valuable at this point?

Reply:
Unit tests should work OTB, but this testing is related to how devs test manually the packages for a full testing procedure. 

It can be removed once every PR starts containing integration tests.